faq/faqs-business-logic.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "FAQs: {{site.bs-bl}}"
+description: "An overview of the most common {{site.tp}} {{site.bs-bl}} operations presented in a Q&A format."
+type: faq
+page_title: "FAQs: Common operations in {{site.bs-bl}}"
+slug: faq-business-logic-data-operations
+position:
+tags: backendservices, cloud-code
+teampulseid:
+ticketid:
+pitsid:
+---
+
+## Environment
+<table>
+  <tr>
+    <td>Service</td>
+    <td>Business Logic</td>
+  </tr>
+  <tr>
+    <td>Feature</td>
+    <td>Cloud Code for Data, Cloud Functions</td>
+  </tr>
+</table>
+
+## Introduction
+
+This article answers questions about common operations with the {{site.bs-bl}} service of {{site.tp}}. They mainly revolve around using the `request` and `response` objects to make application-critical checks and also performing common operations.
```

Comment:
Indeed, the second sentence is a bit unclear - the questions are about common operations and application-critical checks, not especially about the _request_ and _response_ parameters. Re-writing this for more clarity in the introduction.